versions.json

```diff
@@ -2,38 +2,223 @@
     "avro": {
         "commitish": "release-1.11.0",
         "version_string": "1.11.0",
-        "license": "Apache License 2.0",
-        "consortium_build_number": "2",
-        "package_revision": "3",
+        "license": "Apache-2.0",
+        "consortium_build_number": "3",
+        "package_revision": "0",
```

Comment:
How are these related now?

I'm starting to think we should document when to bump/reset these. It seems really easy for someone to mishandle these.

Certainly willing to be persuaded otherwise.

Reply:
Bumping `consortium_build_number` results in a separate package that installs to a separate directory.
Bumping `package_revision` results in a new version of the same package, which will replace the previous version of the package.

Reply:
> installs to a separate directory

meaning, it can live alongside another build of the same package on the same machine.

Reply:
> Bumping `consortium_build_number` results in a separate package that installs to a separate directory.
Bumping `package_revision` results in a new version of the same package, which will replace the previous version of the package.

That makes sense, but we need to write down when one or both of those numbers need to be reset or bumped. The reason for that is primarily for the rest of the team.

The README seems like a fine place for that.

Reply:
I've added a section to the FAQ.